net.sourceforge.Pdfedit.json

```diff
@@ -0,0 +1,173 @@
+                },
+                {
+                    "type": "patch",
+                    "strip-components": 0,
+                    "options": ["-l"],
+                    "paths": [
+                        "fedora_rpm_patches/0079-compositing-types.patch",
+                        "fedora_rpm_patches/0080-net-wm-sync-request-2.patch",
+                        "fedora_rpm_patches/0084-compositing-properties.patch",
+                        "fedora_rpm_patches/qt-3.3.8-libpng15.patch"
+                    ]
+                },
+                {
+                    "type": "patch",
+                    "path": "qt3_flatpak_build_fixes.patch"
+                }
+            ]
+        },
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh",
+                "./b2 install --prefix=/app --with-system --with-random define=\"BOOST_SYSTEM_NO_DEPRECATED\" -sNO_BZIP2=1 cxxflags=-fPIC cflags=-fPIC"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.65.1/source/boost_1_65_1.tar.gz",
+                    "sha256": "a13de2c8fbad635e6ba9c8f8714a0e6b4264b60a29b964b940a22554705b6b60"
+                }
+            ]
+        },
+        {
+            "name": "pdfedit",
+            "buildsystem": "simple",
+            "build-commands": [
+                "QTDIR=/app autoconf",
+                "QTDIR=/app ./configure --prefix=/app --with-boost=/app",
+                "QTDIR=/app /usr/bin/make clean",
+                "mkdir /app/share || echo '/app/share already exists'",
+                "QTDIR=/app CFLAGS=\"$CFLAGS -w -fPIC\" CXXFLAGS=\"$CXXFLAGS -fPIC -w -DQSDEBUGGER -DQT_CLEAN_NAMESPACE -DCONFIG_DATA_PATH=/app/share\" /usr/bin/make -e",
+                "QTDIR=/app /usr/bin/make install"
+            ],
+            "post-install": [
+                "install -Dm644 /app/share/icon/pdfedit_icon.svg /app/share/icons/hicolor/scalable/apps/net.sourceforge.Pdfedit.svg",
+                "for size in 16 32 64 128; do
+                    rsvg-convert -w $size -h $size -f png -o $size.png /app/share/icons/hicolor/scalable/apps/net.sourceforge.Pdfedit.svg
+                    install -Dm644 $size.png /app/share/icons/hicolor/${size}x${size}/apps/net.sourceforge.Pdfedit.png
+                done",
+                "desktop-file-install --dir=/app/share/applications net.sourceforge.Pdfedit.desktop",
+                "install -Dm644 net.sourceforge.Pdfedit.metainfo.xml /app/share/metainfo/net.sourceforge.Pdfedit.metainfo.xml"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://git.code.sf.net/p/pdfedit/git",
+                    "tag": "0.4.5.fl1",
+                    "commit": "0d9c8a9171d12b59ea97eb908db07a06f259a908",
+                    "only-arches": [
```

Comment:
No need to specify it here if you're using flathub.json; also i386 is dropped since 20.08.

Reply:
Ok I drop it.